src/frontends/tensorflow_lite/include/openvino/frontend/tensorflow_lite/extension/conversion.hpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
```

Comment:
let's also add new folder with TF Lite FE headers to labeler for `CPP API` here https://github.com/openvinotoolkit/openvino/blob/master/.github/labeler.yml#L32

And I suggest adding a new label on GitHub for TF Lite and add it to labeler as well (see PDPD, TF as examples)

Reply:
Added label in GH and updated labeler. Thx!